WinUIGallery/WinUIGallery.csproj

```diff
@@ -70,22 +70,15 @@
     <PackageReference Include="System.Private.Uri" />
   </ItemGroup>
   <ItemGroup Condition="'$(IsInWinUIRepo)' != 'true'">
-    <PackageReference Update="Microsoft.WindowsAppSDK" Version="$(WindowsAppSdkPackageVersion)" />
+    <PackageReference Update="Microsoft.WindowsAppSDK" Version="1.4.230628000-preview1" />
```

Comment:
Do we really want to reference it that way now instead of continuing using the build variable for that?